geo/src/test/java/org/odk/collect/geo/geopoly/GeoPolyFragmentTest.kt

```diff
@@ -4,6 +4,7 @@ import android.app.Application
 import android.os.Bundle
 import androidx.fragment.app.FragmentResultListener
 import androidx.lifecycle.Lifecycle
+import androidx.lifecycle.MutableLiveData
```

Comment:
There is a lot of `!!` in kotlin version. Can we review the class and improve that?